api/service/metrics/metrics.pb.go

```diff
@@ -0,0 +1,433 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: metrics.proto
```

Comment:
No need to commit generated file

Reply:
Actually we made the decision to commit generated file: Otherwise `go get github.com/harmony-one/harmony` gets broken.  Go maintainers made a conscious decision not to run any `//go:generate` lines—which would generate `.pb.go` files—on `go build` command internally run by `go get`, arguing (properly) that running arbitrary command line of the source code author's choice is a potential security loophole.  (I mean, `//go:generate rm -rf /`…)

Reply:
@astralblue Interesting point, thanks for sharing!